src/Altinn.Platform/Altinn.Platform.Storage/Storage/Repository/InstanceRepository.cs

```diff
@@ -527,7 +526,7 @@ public async Task<Instance> Update(Instance item)
         {
             PreProcess(item);
 
-            ItemResponse<Instance> instance = await Container.UpsertItemAsync<Instance>(item, new Microsoft.Azure.Cosmos.PartitionKey(item.InstanceOwner.PartyId));
+            ItemResponse<Instance> instance = await Container.UpsertItemAsync<Instance>(item, new PartitionKey(item.InstanceOwner.PartyId));
```

Comment:
In the create call we've changed the return type to be only Instance while here we kept ItemResponse. On purpose?

Reply:
In all cases where we don't process the ItemResponse but only take out the content we can use the simplified return type. Can clean up the implementation so we use Instance as return.